### PR TITLE
update README

### PR DESCRIPTION
**Pull request by MikeLing**
肯定有很多需要吐槽的地方，中英文都是.....还有一些措辞，不知道大家是否觉得恰当。

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,13 @@ You can participate in this activity as long as you are college students over 18
 
 只要是你 18 周岁以上的大学生，就可以参加这个活动，你完全可以把它当作你的暑期实习，并且性价比和含金量都比一般实习还高。
 
+## What's gsoc-cn for?
+
+This organization is made up of previous GSoC student and would like to share their experience about how to apply and pass GSoC in here. If you find someone in here works for the same organization that you want to apply this year, feel free to contact them on [Telegram group](https://t.me/joinchat/B30CMA1YxJ0g0yo3GLj8uA) or IRC channel ([#gsoccn](https://www.irccloud.com/invite?channel=%23gsoccn&hostname=chat.freenode.net&port=6697&ssl=1) on freenode). I believe everyone in here is willing to tell you how to get a GSoC slot by writing a good proposal and negotiate with your community, but please be patient and polite when you ask for help :)
```
Comment (OXPHOS) [inline README.md]: ..GSoC students who would like to share their experience of applying and passing GSoC. If you find members here interned for the organization(s) you want to apply for this year...

File: README.md [hunk 2]
```diff
@@ -10,6 +10,13 @@ You can participate in this activity as long as you are college students over 18
...
+
+这个社区是由往年参加过GSoC的学生组成的，这里的每个人都愿意分享关于如何申请并通过GSoC的经验。如果你发现社区成员中有你想要申请的社区的往届GSoC student，你可以在我们的[Telegram group](https://t.me/joinchat/B30CMA1YxJ0g0yo3GLj8uA) 或者IRC 频道 ([#gsoccn](https://www.irccloud.com/invite?channel=%23gsoccn&hostname=chat.freenode.net&port=6697&ssl=1) on freenode)联系他们，向他们请教关于如何申请的经验和与社区沟通的技巧，但是请在提问的时候保持耐心和礼貌。 :)
+
```
Comment (OXPHOS) [inline README.md]: students

File: README.md [hunk 3]
```diff
@@ -10,6 +10,13 @@ You can participate in this activity as long as you are college students over 18
 
 只要是你 18 周岁以上的大学生，就可以参加这个活动，你完全可以把它当作你的暑期实习，并且性价比和含金量都比一般实习还高。
 
+## What's gsoc-cn for?
+
+This organization is made up of previous GSoC students who would like to share their experience of applying and passing GSoC. If you find members here interned for the organization(s) you want to apply for this year, feel free to contact them on [Telegram group](https://t.me/joinchat/B30CMA1YxJ0g0yo3GLj8uA) or IRC channel ([#gsoccn](https://www.irccloud.com/invite?channel=%23gsoccn&hostname=chat.freenode.net&port=6697&ssl=1) on freenode). I believe everyone in here is willing to tell you how to get a GSoC slot by writing a good proposal and negotiate with your community, but please be patient and polite when you ask for help :)
+
+这个社区是由往年参加过GSoC的学生组成的，这里的每个人都愿意分享关于如何申请并通过GSoC的经验。如果你发现社区成员中有你想要申请的社区的往届GSoC students，你可以在我们的[Telegram group](https://t.me/joinchat/B30CMA1YxJ0g0yo3GLj8uA) 或者IRC 频道 ([#gsoccn](https://www.irccloud.com/invite?channel=%23gsoccn&hostname=chat.freenode.net&port=6697&ssl=1) on freenode)联系他们，向他们请教关于如何申请的经验和与社区沟通的技巧，但是请在提问的时候保持耐心和礼貌。 :)
```
Comment (gaocegege) [inline README.md]: 这里可不可以也中英文空格呢 /w\

Comment (MikeLing) [reply README.md]: ok ;)

**Comment by MikeLing** [comment]
那我merge啦！谢谢大家的review

**Pull request merged**